### PR TITLE
fill policy with domain id for technical users

### PR DESCRIPTION
**Pull request by Carthaca**
@andypf please have a look

### CODE REVIEW

**Comment by andypf** [comment]
lgtm

File: plugins/identity/app/controllers/identity/domains/technical_users_controller.rb [hunk 1]
```diff
@@ -4,17 +4,23 @@ module Identity
   module Domains
     class TechnicalUsersController < ::DashboardController
 
+      before_filter do
+        @policy_default_params ||= {}
+        @policy_default_params[:user] = { domain_id: @scoped_domain_id }
```
Comment (Carthaca) [inline plugins/identity/app/controllers/identity/domains/technical_users_controller.rb]: maybe `@policy_default_params[:user] = current_user` instead ?

Comment (andypf) [inline plugins/identity/app/controllers/identity/domains/technical_users_controller.rb]: no, current_user doesn't have the attribute scoped_domain_id
But there is domain_id and user_domain_id. So you could change the rule in policy

Comment (andypf) [inline plugins/identity/app/controllers/identity/domains/technical_users_controller.rb]: ah sorry. You are right
Use current_user

Comment (ruvr) [inline plugins/identity/app/controllers/identity/domains/technical_users_controller.rb]: why is the user relevant at all in the policy? shouldn't it just check for other cloud_admin or a admin role for the current domain?

**Comment by Carthaca** [comment]
checking for domain_id not being nil is what we want here: i.e. user is domain admin

**Comment by andypf** [comment]
👍

**Pull request merged**